### ISSUE TITLE
[Accessibility]: Update extension button 

### ISSUE THREAD

**Issue by abeatrix** [sourcegraph/sourcegraph#34851]
### Audit type

Keyboard navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/34050

### Problem description

Clicking on the Update extension does not trigger page rerender so the focus will stay on the button after the button has been clicked. It is not cleared to the user if clicking on the button has performed any action:
![update_before](https://user-images.githubusercontent.com/68532117/166562512-2bfe27fa-c2a5-4736-97db-10c1f43b72ec.gif)

![image](https://user-images.githubusercontent.com/68532117/166560719-e1f09672-8422-45aa-8940-387eab2c0bf7.png)

### Expected behavior

Display message to let users know whether the action has been performed successfully 

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [ ] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

**Comment by sourcegraph-bot-2** [comment]
Heads up @muratsu @jjinnii @ryankscott - the "team/integrations" label was applied to this issue.
